docs/topics/Threads.md

```diff
@@ -163,24 +163,42 @@ Additionally, when a client loses access to a channel they are not removed from
-- Threads in a forum have the same permissions behavior as threads in a text channel, inheriting all permissions from the parent channel, with just one exception: Creating a thread in a forum channel only requires the permission that is currently named `SEND_MESSAGES`.
-- The first message in a forum thread can contain additional markdown for bulleted list and headings.
-- A thread can be pinned within a forum. A thread that is pinned will have the `(1 << 1)` flag set. Archiving a pinned thread will unset the flag. A pinned thread will not auto archive.
-- Forums can specify `available_tags` that can be set on individual threads via the `applied_tags` field.
+### Creating Threads in Forum Channels
+
+Threads appear as forum posts within forum channels, and are created the same way as threads in text channels. However, private threads are currently not supported in forum channels so all threads created must be of type `PUBLIC_THREAD`.
+
+Threads can be created using the [`POST /channels/<channel_id>/threads`](#DOCS_RESOURCES_CHANNEL/start-thread-in-forum-channel) endpoint. When creating threads in forum channels, the API will *also* create a message that will have the same ID as the thread. Since a message is being created, you're required to pass parameters for both a message and a thread when calling the create thread endpoint.
+
+Threads in a forum channel have the same permissions behavior as threads in a text channel, inheriting all permissions from the parent channel, with one exception: creating a thread in a forum channel only requires the `SEND_MESSAGES` permission.
+
+### Forum Channel Fields
+
+It's worth calling out a few details about fields specific to forum channels that may be important to keep in mind:
+
+- The `last_message_id` field is the ID of the most recently created thread in that channel. As with messages, your app will not receive a `CHANNEL_UPDATE` event when the field is changed. Instead clients should update the value when receiving [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) events.
+- The `topic` field is what is shown in the "Guidelines" section within the Discord client.
+- The `rate_limit_per_user` field limits how frequently threads can be created. There is a new `default_thread_rate_limit_per_user` field on forums as well, which limits how often messages can be sent _in a thread_. This field is copied into `rate_limit_per_user` on the thread at creation time.
+- The `available_tags` field can be set when creating or updating a channel, which determines which tags can be set on individual threads within the thread's `applied_tags` field.
```

Comment:
Might also be worth pointing out the `REQUIRE_TAG` flag for forum channels here.